app/views/layouts/application.html.erb

```diff
@@ -18,7 +18,7 @@
           </div>
   <%= yield %>
 </div>
-  <script src="/js/jquery.js"></script>
-  <script src="/js/bootstrap.min.js"></script>
+  <!-- <script src="/js/jquery.js"></script> -->
+  <!-- <script src="/js/bootstrap.min.js"></script> -->
```

Comment:
@slague these two scripts are already being loaded via your asset pipeline and included via line 6. this may have been throwing your code off. be sure to delete this completely. it's possible the commented out part isn't working to comment it out!